keyboards/clueboard/66_hotswap/gen1/config.h

```diff
@@ -1,5 +1,4 @@
 #pragma once
-#include "config_common.h"
```

Comment:
This is still needed. 

```suggestion
#include "config_common.h"
```